modernomad/settings/local.py

```diff
@@ -1,16 +1,21 @@
+import sys
 from .common import *  # noqa
-from .common import INSTALLED_APPS
+from .common import INSTALLED_APPS, MIDDLEWARE_CLASSES
 
 DEBUG = True
-INSTALLED_APPS = INSTALLED_APPS + [
-    'debug_toolbar'
-]
+TESTING_MODE = 'test' in sys.argv
```

Comment:
A bikeshed: `RUNNING_TESTS`? Then the statement below becomes `if not RUNNING_TESTS` which reads quite nice.